proto/dolt/services/eventsapi/v1alpha1/event_constants.proto

```diff
@@ -90,6 +90,7 @@ enum ClientEventType {
     STASH_DROP = 58;
     STASH_LIST = 59;
     STASH_POP = 60;
+	SHOW = 61;
```

Comment:
(minor) looks like a little formatting weirdness. Can you check this out and see if it's just an errant tab messing up the indent?

Reply:
Not sure what happened that, but it explains why the protobuf CI check is failing for this PR. It wasn't obvious to me because of the tab width in my editor.